lib/content-style/parser.rb

```diff
@@ -16,6 +16,28 @@ def parse(file_content)
         file_tree
       end
 
+      def get_erb_locations(file_content)
+        hotcop_lines = []
+        hotcop_identifiers = []
+        current_line_number = 1
+        s = StringScanner.new(file_content)
+        while (line_content = s.scan_until(/\n/))
```

Comment:
You can replace the above three lines with:
```ruby
file_content.each_line.with_index do |line_content, current_line_number|
   ....
end
```
`each_line` includes the newline character, so if you want to get rid of it you can do `line_content.chomp`